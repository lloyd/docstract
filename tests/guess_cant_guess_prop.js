/**
 * This is a property, but we can't guess the name!
 * @type {string}
 */

