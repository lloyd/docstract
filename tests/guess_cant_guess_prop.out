{
  "args": [
    "can't determine property name at line 1"
  ], 
  "exception_type": "<type 'exceptions.RuntimeError'>", 
  "stack": [
    [
      "<module>", 
      "got = ds.extractFromFile(os.path.join(testDir, test + \".js\"))"
    ], 
    [
      "extractFromFile", 
      "data = self.extract(contents)"
    ], 
    [
      "extract", 
      "self._analyzeBlock(block, context, firstBlock, stack, lineStart, line)"
    ], 
    [
      "_analyzeBlock", 
      "parseData['blockHandler'].merge(doc, stack[-1][1], guessedName, self._whatContext(stack))"
    ], 
    [
      "merge", 
      "raise RuntimeError(\"can't determine property name\")"
    ]
  ]
}
